api/maven-api-xml/pom.xml

```diff
@@ -27,7 +27,8 @@
   </parent>
 
   <artifactId>maven-api-xml</artifactId>
-  <name>Maven API XML</name>
+  <name>Maven 4 API XML</name>
```

Comment:
I think both name and artifactId are too generics. `maven-api-xml-helpers`, etc. fully describe the intent

Reply:
It's not helpers, it contains a single class which is the XML API used by maven:
  https://maven.apache.org/ref/4-LATEST/apidocs/org/apache/maven/api/xml/package-summary.html